### PR TITLE
Project documentation

### PR DESCRIPTION
**Pull request by hendridg**
# Link's of videos

[Link of page presentation](https://www.loom.com/share/1c28d7b2fc614a2b856dac665adf03f3?sharedAppSource=personal_library)
[Link of Visual Studio video explanation my favorite code part](https://www.loom.com/share/f2bda51f54854380807971b034ccf3fb?sharedAppSource=personal_library)

> I grab two videos because my OS Fedora, don't let me share my full screen in Loom, Thanks for your understanding.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -14,6 +14,14 @@ Portfolio to Microverse course
...
+![screenshot](./assets/screenshots/screenshot_8.png)
+![screenshot](./assets/screenshots/screenshot_9.png)
+![screenshot](./assets/screenshots/screenshot_10.png)
+![screenshot](./assets/screenshots/screenshot_11.png)
+![screenshot](./assets/screenshots/screenshot_12.png)
+![screenshot](./assets/screenshots/screenshot_13.png)
+![screenshot](./assets/screenshots/screenshot_14.png)
+![screenshot](./assets/screenshots/screenshot_15.png)
 
 ## Built With
```
Comment (ElijahTobs) [inline README.md]: - [ ] The README is expected to contain an "Acknowledgements" section or similar that gives appropriate credit to Cindy Shin, the author of the original design, as required by the Creative Commons License.

File: README.md [hunk 2]
```diff
@@ -14,6 +14,14 @@ Portfolio to Microverse course
...
 ![screenshot](./assets/screenshots/screenshot_6.png)
 ![screenshot](./assets/screenshots/screenshot_7.png)
+![screenshot](./assets/screenshots/screenshot_8.png)
+![screenshot](./assets/screenshots/screenshot_9.png)
+![screenshot](./assets/screenshots/screenshot_10.png)
+![screenshot](./assets/screenshots/screenshot_11.png)
+![screenshot](./assets/screenshots/screenshot_12.png)
+![screenshot](./assets/screenshots/screenshot_13.png)
+![screenshot](./assets/screenshots/screenshot_14.png)
+![screenshot](./assets/screenshots/screenshot_15.png)
```
Comment (ElijahTobs) [inline README.md]: - [ ] __[OPTIONAL]__ Two or at most three screenshots of your project is fine

**Review by ElijahTobs — CHANGES_REQUESTED** [review]
# Status: Changes Required ⚠️ 🚦
## Hi @hendridg  👋

You've done really well so far! 👍 

## Major Highlights 💯 

Your README is professional ✔️ 
You've followed the Github flow rules. ✔️ 
You have good commit messages ✔️ 

_However, there are some changes that you are required to make before this project can be approved._

## Required Changes ♻️

- [ ] The content of the footer should be horizontally-centred and not aligned to the top.

Yours | Original
--- | ---
image | image


_See the inline comments below_

## Optional Changes 👌
_Every comment with the OPTIONAL prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend that you take them into account as they can make your code better._

___Check for changes marked as Optional in the inline comments below.___


_If anything doesn't seem clear to you, feel free to leave your question or comment in the PR thread below_

#### "Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise."

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
## Happy Coding :computer: 

_Reviewed by_  **Elijah O. Ayandokun**

**Review by rotimiazeez — APPROVED** [review]
# Hi @ElijahTobs 👋, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**